composer.json

```diff
@@ -110,6 +110,9 @@
     "typo3-cms-scripts": [
       "typo3cms install:fixfolderstructure",
       "typo3cms install:generatepackagestates"
+    ],
```

Comment:
Could you provide the explanation, why we need it? Thanks in advance!

Reply:
Reverted to upstream master